@xen-orchestra/lite/src/stories/web-core/tree-view/tree-list.story.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <ComponentStory :params="[]">
+    <TreeList>
+      <TreeItem>
+        <TreeItemLabel route="dashboard" :icon="faCity"> Pool </TreeItemLabel>
+        <template #sublist>
+          <TreeList v-for="i in 3" :key="i">
+            <TreeItem>
```

Comment:
I think `v-for` should be on `TreeItem`:

```suggestion
          <TreeList>
            <TreeItem v-for="i in 3" :key="i">
```